### ISSUE TITLE
VC1 files played at fast speed

### ISSUE THREAD

**Issue by Oibaf66** [xbmc/xbmc#15146]
### Describe the bug
Here is a clear and concise description of what the problem is:

The files with VC1 codec are played at too high speed and the sound goes out of sync. 
This happens with Mi box 3 (Oreo build 2396) and Leila RC3.
The issue is not present with Kripton 17.6.
All the other codecs are played properly (H263, H264, MPEG-4, WMV9, VP9, etc.).

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

The files should be played at normal speed.

## Actual Behavior

The video is rendered at fast speed.

### To Reproduce

1. Play https://drive.google.com/file/d/0BwxFVkl63-lERnhlY1pHeW5wbjQ/view?usp=sharing with Mi box 3 and Leila RC3

### Debuglog

The debuglog can be found here:

https://paste.kodi.tv/ukewirafob.kodi

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [ X] Android
 - [ ] iOS
 - [ ] Linux
 - [ ] OSX
 - [ ] Raspberri-Pi
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Oreo (build 2396)
 - Kodi version: Leila RC3

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by KarellenX** [comment]
Please post your log to a patebin type site. Use https://paste.kodi.tv/

We don't like, and there is extra work, downloading unfamiliar files.

**Comment by Oibaf66** [comment]
Pasted the debug log in:

https://paste.kodi.tv/ukewirafob.kodi

**Comment by Oibaf66** [comment]
With Kodi 17.6 the VC1 files are played properly.
Maybe the debug log of the same video played with the same hardware and OS but with KodiI Kripton 17.6 can be usefull: 
https://paste.kodi.tv/seyazocaki.kodi

**Comment by Oibaf66** [comment]
The bug is fixed in Leia RC4

**Comment by IcefingersFr** [comment]
Hi,
I use kodi leia 18.1 on android TV 8.1 Mi BOX S
 today. i've launch some video files in VC-1. 

and i have issue for reading video file : video files seam's reading too fast. 

in my mind, everything works great before 18.1 leila. 

thks for your help

**Comment by KarellenX** [comment]
@IcefingersFr 

Please seek support at the Kodi forums... https://forum.kodi.tv/

Github is not for support.

**Comment by Oibaf66** [comment]
I have the same issue with MI box 3.
I can confirm that with kodi leia 18.1 some (not all) VC-1 files are played at too fast speed and the video and sound go out of sync. 
It seems a bug introduced with Leila.
This is the log:
https://hastebin.com/xonowiruyo.sql

**Comment by IcefingersFr** [comment]
@ KarellenX  
Hi, maybe i did not clearly express myself : i don't want any support here... 
i just would like inform you, since the patch for VC-1 codec playback included in RC4. there have been some update of KODI. and during theses updates the fix for VC-1 video codec playback, no longer work.

since kodi 18.1, i have exactly the same problem as @Oibaf66  .

I specify all work well with kodi 17 Krypton.

in conclusion : @Oibaf66 u are not alone with your problem. the problem is not come from your device or whatever .. the probleme is only a bug in the code.. 
that's why i would like inform githubs and not a regular support forum.

thks.

sorry to try to help the community with feedback.

have a nice evening, team.

**Comment by DaVukovic** [comment]
No further response from the op, no clean log

Issue closed
